sections/common.md

```diff
@@ -194,6 +194,22 @@ The parsing type can be set on the service level and it is recommended to only s
 
 See the chapter on <a href="https://github.com/Intelecom/sms/blob/master/sections/about.md#message-content">message content</a> for further details about the different parsing types.
 </td></tr>
+<tr><td>strex_verification_timeout</td><td>String</td><td>Used for Strex verification process. Specified time to wait for a end user Strex verification before timing out. </br></br> Valid Values: 0-30</td></tr> 
```

Comment:
Suggested re-write: "Specified time in minutes to wait for an end user to complete verification before timing out. "

Reply:
Yup, sounds better. Will change.